content-security-policy/connect-src/connect-src-xmlhttprequest-blocked.sub.html

```diff
@@ -17,17 +17,13 @@
             log("violated-directive=" + e.violatedDirective);
         });
 
-        try {
-            var xhr = new XMLHttpRequest;
-            xhr.open("GET", "http://www1.{{host}}:{{ports[http][0]}}/content-security-policy/support/fail.png", true);
-            xhr.send();
-            xhr.onload = function() {
-                log("Fail");
-            }
-            xhr.onerror = function() {
-                log("Pass");
-            }
-        } catch (e) {
+        var xhr = new XMLHttpRequest;
+        xhr.open("GET", "http://www1.{{host}}:{{ports[http][0]}}/content-security-policy/support/fail.png", true);
+        xhr.send();
+        xhr.onload = function() {
+            log("Fail");
+        }
```

Comment:
Can't we use the assert unreached stuff here? Oh, maybe this is using its own test nonsense? Sigh.

Reply:
log("FAIL") uses assert_unreached under the hood.